depends/packages/qt.mk

```diff
@@ -178,9 +178,9 @@ define $(package)_preprocess_cmds
   patch -p1 -i $($(package)_patch_dir)/no-xlib.patch &&\
   echo "QMAKE_LINK_OBJECT_MAX = 10" >> qtbase/mkspecs/win32-g++/qmake.conf &&\
   echo "QMAKE_LINK_OBJECT_SCRIPT = object_script" >> qtbase/mkspecs/win32-g++/qmake.conf &&\
-  sed -i.old "s|QMAKE_CFLAGS            = |!host_build: QMAKE_CFLAGS            = $($(package)_cflags) $($(package)_cppflags) |" qtbase/mkspecs/win32-g++/qmake.conf && \
```

Comment:
FWIW if you use bash as shell, you need to escape the `!` here to not get a `-bash: !host_build: event not found` error
(I tried executing this command separately to test it)

Reply:
Should we explicitly set `SHELL` then? The default is `/bin/sh` I think.

Reply:
Ah gotcha, fun fact: when `argv[0]` looks like it's `sh`, I believe `bash` will be in posix-compliant mode.